moto/dynamodb2/models.py

```diff
@@ -285,6 +285,9 @@ def __init__(self, hash_key, hash_key_type, range_key, range_key_type, attrs):
     def __repr__(self):
         return "Item: {0}".format(self.to_json())
 
+    def size(self):
+        return sum(bytesize(key) + value.size() for key, value in self.attrs.items())
```

Comment:
Is it good or a standard practice in this project to comment with a link to the oficial docs ?